src/debug/telemetry/Kconfig

```diff
@@ -9,3 +9,12 @@ config SOF_TELEMETRY
 	  systick_info measurement which measures scheduler task performance and may
 	  slightly affect overall performance.
 
+config SOF_TELEMETRY_PERFORMANCE_MEASUREMENTS
+	bool "enable performance measurements"
+	default n
+	help
+	  Enables performance measurements. Each created component can have its
+	  performance checked by measuring execution time of copy function. Performance
+	  records are stored in the limited number of slots in Memory Window 3, so only a
+	  certain number (PERFORMANCE_DATA_ENTRIES_COUNT) of components can be measured.
+
```

Comment:
So not sure what to do. This text again suggests this is a generic capability, while in practise this can be only used on platforms that implement the adsp-debug.window.h interface (i.e. only Intel platforms now).

Reply:
Changed the text, should we change the basic telemetry one as well?